### PR TITLE
Add test coverage for firewalls and firewall-editor

### PR DESCRIPTION
**Pull request by ykim-1**
## 📝 Description

[TPT-1762](https://jira.linode.com/browse/TPT-1762) 
Originally meant to add tests for firewall-editor, however due to the way that firewall-editor opens up a termios terminal in the code, the plugin cannot be opened via process/subprocess in python. Hence, I have decided to create tests for covering use cases of the editor using `linode-cli firewalls rules-update` which is essentially what the firewall-editor calls when we enter the prompt via stdin

## ✔️ How to Test

`make testint`

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

**Review by lgarber-akamai — CHANGES_REQUESTED** [review]

**Comment by lgarber-akamai** [comment]
/acctest sha=fb6307d98613d849ff8faac7f38ff906f5cc7b95 tests=firewalls

File: tests/integration/firewalls/test_firewalls_rules.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            .stdout.decode()
+            .rstrip()
+        )
+    except:
+        logging.exception("Failed creating domain in setup")
+
+    yield firewall_id
+    # teardown - delete all firewalls
+    try:
+        remove_all(target="firewalls")
```
Comment (zliang-akamai) [inline tests/integration/firewalls/test_firewalls_rules.py:46]: Similar issue here

Comment (ykim-1) [reply tests/integration/firewalls/test_firewalls_rules.py]: Fixed! will put up a PR soon doing this for all of the test suites to limit the deletion to only resources created during test execution

**Comment by lgarber-akamai** [comment]
/acctest sha=2dd38f041407058e61186a54eeb804b9725f5bd2 tests=firewalls

**Review by lgarber-akamai — APPROVED** [review]
Looks good!

**Pull request merged**